### PR TITLE
Address fix

### PR DESCRIPTION
**Pull request by descorp**
### Changes

* fix minimum length for house number on Address form
* support UK addresses

### CODE REVIEW

File: Tests/AdyenTests/Card Tests/CardComponentTests.swift [hunk 1]
```diff
@@ -762,26 +762,7 @@ class CardComponentTests: XCTestCase {
             XCTAssertEqual(postalCodeItemView!.titleLabel.text, "Postal code")
             XCTAssertEqual(headerItemView!.text, "Billing address")
 
-            XCTAssertTrue(houseNumberItemView!.alertLabel.isHidden)
```
Comment (descorp) [inline Tests/AdyenTests/Card Tests/CardComponentTests.swift:765]: No need for this. It is already tested above

**Review by erenbesel — APPROVED** [review]

File: Adyen/UI/Form/Items/Address/AddressViewModel.swift [hunk 2]
```diff
@@ -60,7 +60,7 @@ internal struct AddressViewModel {
         case "GB":
             viewModel.labels[.city] = .cityTownFieldTitle
             viewModel.placeholder[.city] = .cityTownFieldPlaceholder
-            viewModel.schema = [.houseNumberOrName, .street, .city, .postalCode, .stateOrProvince].map { .item($0) }
+            viewModel.schema = [.houseNumberOrName, .street, .city, .postalCode].map { .item($0) }
```
Comment (mohammedDehairy) [inline Adyen/UI/Form/Items/Address/AddressViewModel.swift:63]: why the schema change?

Comment (descorp) [reply Adyen/UI/Form/Items/Address/AddressViewModel.swift:63]: UK doesn't have province.

Comment (mohammedDehairy) [reply Adyen/UI/Form/Items/Address/AddressViewModel.swift:63]: in that case, is it added for example for US dynamically?

Comment (descorp) [reply Adyen/UI/Form/Items/Address/AddressViewModel.swift:63]: it is there by default, just hidden for UK.
For US and Brazil it is a dropDown

Comment (mohammedDehairy) [reply Adyen/UI/Form/Items/Address/AddressViewModel.swift:63]: 👍

**Review by mohammedDehairy — APPROVED** [review]

**Pull request merged**